### PR TITLE
[GROG-TECH] new cocktails section

### PR DESCRIPTION
**Pull request by anneconover**
Add cocktails section so that greg may accept fun cocktail recipes that the team creates

### CODE REVIEW

**Comment by anneconover** [comment]
image
image

**Review by rebeccasiow — APPROVED** [review]

File: src/pages/cocktails/tjs-marg.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+Easy and delicious margarita for any day.
+
+<!-- excerpt end -->
+
+## Ingredients
+
+- Casamigos Tequila
+- Ice
+- Trader Joes Jalepeno Limeade
```
Comment (rebeccasiow) [inline src/pages/cocktails/tjs-marg.md]: jalapeno

File: src/pages/cocktails/tjs-marg.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## Directions
+
+### Step 1
+
+Put ice and tequila in cup
+
+### Step 2
+
+Put limade in cup
```
Comment (rebeccasiow) [inline src/pages/cocktails/tjs-marg.md]: yikes

File: src/pages/cocktails/tjs-marg.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+### Step 3
+
+enjoy
```
Comment (rebeccasiow) [inline src/pages/cocktails/tjs-marg.md:30]: nice

**Review by rebeccasiow — APPROVED** [review]

**Comment by rebeccasiow** [comment]
so excited for this feature @groman00 !!!! when will this be released????????????

**Comment by rebeccasiow** [comment]
the romanwebnetizens are WAITING

File: src/pages/cocktails/index.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    {/*
+    https://www.gatsbyjs.org/docs/adding-analytics/
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-112401659-1"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+      gtag('config', 'UA-112401659-1');
+    </script>
+    */}
```
Comment (groman00) [inline src/pages/cocktails/index.tsx:19]: I'm pretty sure i added google analytics somewhere else.  Please delete!

**Review by groman00 — CHANGES_REQUESTED** [review]
Yea i think cocktails should be a subcategory........

- Recipes
-- Dinners
-- Cocktails
-- Desserts

good luck

**Comment by rebeccasiow** [comment]
is this an interview question

**Pull request closed without merging**